src/App.tsx

```diff
@@ -15,48 +24,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  sortFiled: SortType,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortFiled) {
+    preparedGoods.sort((goodA, goodB) => {
+      switch (sortFiled) {
+        case SortType.Alphabet:
+          return goodA.localeCompare(goodB);
+
+        case SortType.Length:
+          return goodA.length - goodB.length;
+
+        default:
+          return 0;
```

Comment:
Would be great to move the switch out of the sort callback, so you don't need to call sort if it's a default case